pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -339,6 +341,18 @@ func (v *vpnSeedServer) Deploy(ctx context.Context) error {
 									Value: v.podNetwork,
 								},
 							},
+							ReadinessProbe: &corev1.Probe{
+								Handler: corev1.Handler{
+									Exec: &corev1.ExecAction{
+										Command: []string{
+											"nc",
+											"-z",
+											"127.0.0.1",
+											fmt.Sprintf("%d", openVpnPort),
+										},
+									},
```

Comment:
You can use a TCP probe, is there any reason a command probe was implemented?
```suggestion
									TCPSocket: &corev1.TCPSocketAction{
										Port: intstr.FromInt(openVpnPort),
									},
```

Reply:
it was implemented using netcat as I reused the probe from a public open vpn helm chart. However, unlike a generic open vpn deployment our's does not need to support udp. Therefore, a tcp probe indeed is much more concise.
I adapted the change accordingly in the latest commit.